src/instruments/src/EFB/TODCalculator/Data/Data.tsx

```diff
@@ -63,7 +63,7 @@ const Data = ({
         }
 
         setTodData({ calculation: { input: syncedInput, type: calculationType } });
-    }, [calculationInputSyncEnabled, distance, verticalSpeed, pitchAngle, calculationType, inputValid, syncedInput]);
+    }, [calculationInputSyncEnabled, distance, verticalSpeed, pitchAngle]);
```

Comment:
I'm pretty sure this would also have been fixed by memoizing `syncedInput` and `inputValid`.